config.template.toml

```
@@ -24,6 +24,9 @@ webhook_avatar_url = "" # optional
 runner = 123456789 # optional: sets the webhook avatar url (will be overridden by the webhook_avatar_url attribute)
 level = 20
 
+[SUGGESTIONS] # optional
+url = "" # Webhook URL - optional
```

The key isn't optional, the parent key is.
Also make the config keys sort-of sane by naming;-
```suggestion
webhook_url = ""
```